src/journal/lookup3.c

```diff
@@ -947,10 +947,8 @@ void driver3()
       x = hashlittle(b, len, (uint32_t)1);
       y = hashlittle(b, len, (uint32_t)1);
       if ((ref != x) || (ref != y))
-      {
         printf("alignment error: %.8x %.8x %.8x %d %d\n",ref,x,y,
                h, i);
-      }
```

Comment:
We keep that file intact as far as possible to make it easier to import another upstream version in the future. Please drop this chunk. (It doesnt' follow our style anyway.)

Reply:
Added an exception for this file.